docs/release-notes/1682.feat.md

```diff
@@ -0,0 +1 @@
+A new {mod}`anndata.io` module contains all `read_*` and `write_*` functions, and all imports of such functions should go through this module.  Old ways of importing these functions i.e., `from anndata import read_csv` or `from anndata._io.specs import read_elem` will still work, but are now considered deprecated and give a warning on import. with the exception of {func}`anndata.io.read_zarr` and {func}`anndata.io.read_h5ad`, which will remain at the top-level `anndata` without warning.  However, they will be eventually deprecated, and will start to a throw a warning in a future release. `user`{ilan-gold}
```

Comment:
Please also max. one line per sentence.

> However, they will be eventually deprecated, and will start to a throw a warning in a future release.

is that so?

Reply:
I would like to do so but no need to state it here since it isn't even happening yet.  Conversation for another time, good point.